docs/tutorial/batch_predict/batch_predict.ipynb

```diff
@@ -0,0 +1,1042 @@
+{
```

Comment:
<p>I don't think we should install mlrun on every demo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/2415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>